composer.json

```diff
@@ -52,6 +51,8 @@
             "Liip\\FunctionalTestBundle\\": ""
         }
     },
+    "minimum-stability": "beta",
```

Comment:
rc? (not sure that works)

Reply:
`RC`. I used `beta` before because I was required hautelook.